### PR TITLE
CC-14662 Project code migration

### PR DESCRIPTION
**Pull request by kkicheglovspryker**
Ticket: https://spryker.atlassian.net/browse/CC-14662

### CODE REVIEW

**Review by sadkivskyi — APPROVED** [review]

File: src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+use Spryker\Glue\Kernel\AbstractPlugin;
+use SprykerEco\Glue\UnzerRestApi\UnzerRestApiConfig;
+
+class UnzerNotificationResourceRoutePlugin extends AbstractPlugin implements ResourceRoutePluginInterface
+{
+    /**
+     * @var string
+     */
+    private const CONTROLLER_NAME = 'unzer-notification';
+    /**
```
Comment (romanhavrylko) [inline src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php:23]: Please add an empty line.

File: src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+use Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRoutePluginInterface;
+use Spryker\Glue\Kernel\AbstractPlugin;
+use SprykerEco\Glue\UnzerRestApi\UnzerRestApiConfig;
+
+class UnzerNotificationResourceRoutePlugin extends AbstractPlugin implements ResourceRoutePluginInterface
+{
+    /**
+     * @var string
+     */
+    private const CONTROLLER_NAME = 'unzer-notification';
```
Comment (romanhavrylko) [inline src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php]: Shouldn't constants be protected?

File: src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php (hunk 1 above)
Comment (kkicheglovspryker) [reply src/SprykerEco/Glue/UnzerRestApi/Plugin/UnzerNotificationResourceRoutePlugin.php:23]: done but not mandatory

File: codeception.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+paths:
+  tests: tests
+  support: .
+  log: tests/_output
+  data: tests/_data
+  envs: tests/_envs
+
+settings:
+  bootstrap: _bootstrap.php
```
Comment (michbeck) [inline codeception.yml:14]: Can't this be removed while added on line 24 as well?

**Review by michbeck — COMMENTED** [review]
Please see comments ...

**Review by romanhavrylko — APPROVED** [review]

File: .scrutinizer.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+build:
+  environment:
+    php: '7.3'
```
Comment (dmytro-sokolovskyi) [inline .scrutinizer.yml]: Use `7.4`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# UnzerRestApi Module
+[![CI](https://github.com/spryker-eco/unzer-rest-api/actions/workflows/ci.yml/badge.svg)](https://github.com/spryker-eco/unzer-rest-api/actions/workflows/ci.yml)
+[![Latest Stable Version](https://poser.pugx.org/spryker-eco/unzer-rest-api/v/stable.svg)](https://packagist.org/packages/spryker-eco/unzer-rest-api)
+[![License](https://img.shields.io/github/license/spryker-eco/unzer-rest-api.svg?b=master)](https://github.com/spryker-eco/unzer-rest-api)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/build-status/master)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg)](https://php.net/)
```
Comment (dmytro-sokolovskyi) [inline README.md]: `7.3` -> `7.4`

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "spryker-eco/unzer-rest-api",
+    "type": "library",
+    "description": "Unzer PSP Glue module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.3",
```
Comment (dmytro-sokolovskyi) [inline composer.json]: `7.3` -> `7.4`

File: src/SprykerEco/Glue/UnzerRestApi/UnzerRestApiDependencyProvider.php [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    /**
+     * @param \Spryker\Glue\Kernel\Container $container
+     *
+     * @return \Spryker\Glue\Kernel\Container
+     */
+    public function provideDependencies(Container $container): Container
+    {
+        parent::provideDependencies($container);
+
+        $this->addUnzerClient($container);
```
Comment (dmytro-sokolovskyi) [inline src/SprykerEco/Glue/UnzerRestApi/UnzerRestApiDependencyProvider.php:33]: Please, add `spryker-eco/unzer` dependency into `composer.json`

File: .scrutinizer.yml (hunk 4 above)
Comment (michbeck) [reply .scrutinizer.yml]: Fixed

File: README.md (hunk 5 above)
Comment (michbeck) [reply README.md]: Fixed

File: composer.json (hunk 6 above)
Comment (michbeck) [reply composer.json]: Fixed

File: src/SprykerEco/Glue/UnzerRestApi/UnzerRestApiDependencyProvider.php (hunk 7 above)
Comment (michbeck) [reply src/SprykerEco/Glue/UnzerRestApi/UnzerRestApiDependencyProvider.php:33]: Fixed

**Review by michbeck — APPROVED** [review]

**Review by dmytro-sokolovskyi — APPROVED** [review]

**Pull request merged**